internal/scripts/installation/compile_MET_all.sh

```diff
@@ -563,13 +596,17 @@ if [ $COMPILE_NETCDF -eq 1 ]; then
   echo "cd `pwd`"
   run_cmd "./configure --prefix=${LIB_DIR} CFLAGS=-fPIC CXXFLAGS=-fPIC LDFLAGS=-L${LIB_DIR}/lib CPPFLAGS=-I${LIB_DIR}/include > netcdf-c.configure.log 2>&1"
   run_cmd "make ${MAKE_ARGS} install > netcdf-c.make_install.log 2>&1"
-
+  
   echo
   echo "Compiling NetCDF-CXX at `date`"
   tar -xzf ${TAR_DIR}/netcdf-cxx*.tar.gz -C ${LIB_DIR}/netcdf
   cd ${LIB_DIR}/netcdf/netcdf-cxx*
   echo "cd `pwd`"
-  run_cmd "./configure --prefix=${LIB_DIR} LDFLAGS=-L${LIB_DIR}/lib CPPFLAGS=-I${LIB_DIR}/include > netcdf-cxx.configure.log 2>&1"
+  if [[ $machine == "Mac" ]]; then 
```

Comment:
This could be simplified with a variable:

```
  configure_args=""
  if [[ $machine == "Mac" ]]; then 
    configure_args="LIBS=\"${LIBS} -lhdf5_hl -lhdf5 -lz\""
  fi
  run_cmd "./configure --prefix=${LIB_DIR} LDFLAGS=-L${LIB_DIR}/lib CPPFLAGS=-I${LIB_DIR}/include LIBS=\"${LIBS} ${configure_args}\"> netcdf-cxx.configure.log 2>&1"
```

Reply:
Thanks @georgemccabe for this simplification!  I'll change the code and retest to ensure the functionality still works.  I'll follow up.